src/Manager/DecideManager.php

```diff
@@ -22,9 +22,9 @@ class DecideManager extends ManagerBase
     public function decide(Decide $decide)
     {
         $body = $decide->json();
-        $url = '/capture';
+        $url = '/decide';
```

Comment:
I say no. I am about to add some tests for this ticket so I will cover this "/decide" as well.